File: mobile-edge/src/contract/shipment/tracking/ShipmentTrackingSummary.ts
// Marketing America Corp. Oleksandr Tishchenko
export type ShipmentTrackingSummary = {
  shipmentId: string;
  shipmentNumber: string;
  carrierCode: string;
  trackingNumber: string;
  currentStatusCode: string;
  updatedAtIso: string;
};

File: mobile-client/android/Contract/Shipment/Tracking/ShipmentTrackingSummary.kt
package app.mobiling.client.client.contract.shipment.tracking

/**
 * Marketing America Corp. Oleksandr Tishchenko
 */
data class ShipmentTrackingSummary(
    val shipmentId: String,
    val shipmentNumber: String,
    val carrierCode: String,
    val trackingNumber: String,
    val currentStatusCode: String,
    val updatedAtIso: String,
)
